test/functional/inabox/test-utils.js

```diff
@@ -16,14 +16,27 @@
 
 import {Deferred} from '../../../src/utils/promise';
 import {Services} from '../../../src/services';
-import {registerIniLoadListener} from '../../../src/inabox/utils';
+import {getA4AId, registerIniLoadListener} from '../../../src/inabox/utils';
 
-describes.fakeWin('inabox-utils', {}, env => {
+describes.realWin('inabox-utils', {}, env => {
   let getResourcesInRectStub;
   let dispatchEventStub;
   let parentPostMessageStub;
   let initCustomEventStub;
   let ampdoc;
+  let a4aIdMetaElement;
+
+  function addA4AMetaTagToDocument() {
+    a4aIdMetaElement = env.win.document.createElement('meta');
```

Comment:
you can use: dom.js#createElementWithAttributes

Reply:
Done.